test/DotNest.Core.SDK.Tests.UI/Readme.md

```diff
@@ -0,0 +1,11 @@
+# DotNest Core SDK UI Tests
+
+## Overview
+
+This project contains a sample UI test for the DotNest Core SDK, which ensures that basic functionality is working. You can use it as a starting point for your own UI tests. The project uses [Lombiq.Tests.UI module](https://github.com/Lombiq/UI-Testing-Toolbox/blob/dev/Readme.md) which gives you a base for writing UI tests. For more info please check out the [Lombiq.Tests.UI repository](https://github.com/Lombiq/UI-Testing-Toolbox/blob/dev/Readme.md). For more UI test examples see the [Lombiq.Tests.UI.Samples repository](https://github.com/Lombiq/UI-Testing-Toolbox/blob/dev/Lombiq.Tests.UI.Samples/Readme.md).
+
+## Contributing and support
+
+Bug reports, feature requests, comments, questions, code contributions and love letters are warmly welcome. You can send them to us via GitHub issues and pull requests. Please adhere to our [open-source guidelines](https://lombiq.com/open-source-guidelines) while doing so.
+
+This project is developed by [Lombiq Technologies](https://lombiq.com/). Commercial-grade support is available through Lombiq.
```

Comment:
The equivalent of this should always be in the root Readme, which it is, so this can be removed.